src/cli.rs

```diff
@@ -237,35 +298,10 @@ pub fn run() -> Result<()> {
     }
 
     trace!("rekey? {}", opts.rekey);
+    trace!("rekey_all? {}", opts.encrypt_in_place);
```

Comment:
Quick skim made me notice this: should probably be `opts.rekey_all`.

Reply:
Yeah, it should; fixed that.